source/tyr/tests/integration/datapost_test.py

```diff
@@ -32,12 +36,33 @@ def test_post_pbf(create_instance_fr):
     os.remove('/tmp/empty_pbf.osm.pbf')
     assert (not os.path.isfile('/tmp/empty_pbf.osm.pbf'))
 
+
 def test_post_no_file(create_instance_fr):
     tester = app.test_client()
     resp = tester.post('/v0/jobs/fr')
     assert resp.status_code == 404
 
+
 def test_post_bad_instance(create_instance_fr):
     tester = app.test_client()
     resp = tester.post('/v0/jobs/fr_ko')
     assert resp.status_code == 404
+
+
+def test_post_pbf_autocomplete(create_instance_fr):
+    assert (not os.path.isfile('/tmp/empty_pbf.osm.pbf'))
```

Comment:
I've seen that this is done in  the other tests, but I don't like this hardcoded path to tmp, I would prefer to use pytest's `tmpdir` (I'm not sure it's possible though)